modules/hitlnext/src/backend/repository.ts

```diff
@@ -464,4 +472,34 @@ export default class Repository {
       { count: conditions.limit, sortOrder: [{ column: 'id', desc: true }] }
     )
   }
+
+  deleteHandoff = async (id: string) => {
+    return this.bp.database.transaction(async trx => {
+      const handoff = await this.getHandoff(id)
+
+      // We want delete the events since it contains
+      // the user's conversation.
```

Comment:
Useless comment using two lines... Make the code clear about what happens. Less comments = better